src/extension-runners/chromium.js

```diff
@@ -40,6 +40,11 @@ const log = createLogger(__filename);
 
 const asyncMkdirp = promisify(mkdirp);
 
+export const DEFAULT_CHROME_FLAGS = ChromeLauncher.defaultFlags()
+  .filter((flag) => {
+    return flag !== '--disable-extensions' && flag !== '--mute-audio';
```

Comment:
Nit, we may define another const for the chrome flags that we want to filter out, so that we can just add or remove flags from that list based on the issue we may notice (without having to change this filter manually).

@jezhou how that sounds to you?

Reply:
@rpl Sounds good – added a const in b77f064

Reply:
Thanks!